Improve the wording about where to download latest (stable) vast release (#1999)
tenzir/vast: README.md

## Get VAST

We offer pre-packaged versions of VAST for download:

> - **stable*: see the artifacts of the latest official [VAST release][latest-release] 

Reviewer: ```suggestion
- **stable**: see the artifacts of the latest official [VAST release][latest-release] 
```

> Linux users can download our [latest static
> build](https://storage.googleapis.com/tenzir-public-data/vast-static-builds/vast-static-latest.tar.gz)
> based on maser via browser or cURL. The latest stable version can be found on our
> [releases][release-url] page.

Reviewer 2: ```suggestion
We offer pre-packaged versions of VAST for download:
- **stable*: see the artifacts of the latest official [VAST release][latest-release] 
- **development**: we offer a [static
build](https://storage.googleapis.com/tenzir-public-data/vast-static-builds/vast-static-latest.tar.gz) of the master branch for Linux
```

[contributing-url]: https://github.com/tenzir/.github/blob/master/contributing.md
[since-release-badge]: https://img.shields.io/github/commits-since/tenzir/vast/latest.svg?color=green
[since-release-url]: https://github.com/tenzir/vast/compare/2021.06.24...master

> [release-url]: https://github.com/tenzir/vast/releases

Reviewer 2: ```suggestion
[latest-release]: https://github.com/tenzir/vast/releases/latest
```
